charts/drupal/templates/_helpers.tpl

```diff
@@ -412,16 +410,28 @@ fi
 {{- end }}
 
 {{- define "drupal.import-reference-db" -}}
-if [ -f /app/reference-data/db.sql.gz ]; then
+if [[ -f /app/reference-data/db.tar.gz || -f /app/reference-data/db.sql.gz ]]; then
   echo "Dropping old database"
   drush sql-drop -y
 
   echo "Importing reference database dump"
-  gunzip -c /app/reference-data/db.sql.gz > /tmp/reference-data-db.sql
-  pv -f /tmp/reference-data-db.sql | drush sql-cli
+  app_ref_data=/app/reference-data
+  tmp_ref_data=/tmp/reference-data
+
+  # New way of importing.
+  if [[ -f "${app_ref_data}"/db.tar.gz ]]; then
+    mkdir "${tmp_ref_data}"
+    tar -xzf "${app_ref_data}"/db.tar.gz -C "${tmp_ref_data}"/
+    find "${tmp_ref_data}"/ -type f -name "*.sql" | xargs -P10 -I{} sh -c 'echo "Importing {}" && mysql -A --user="${DB_USER}" --password="${DB_PASS}" --host="${DB_HOST}" "${DB_NAME}" < {}'
+
+  # Backwards compatibility for old way of importing.
+  elif [[ -f "${app_ref_data}"/db.sql.gz ]]; then
+    gunzip -c "${app_ref_data}"/db.sql.gz > "${tmp_ref_data}"-db.sql
+    pv -f "${tmp_ref_data}"-db.sql | drush sql-cli
```

Comment:
though the `pv` is good, and `pv -f` is forcing output even if not in the terminal... the output appears only when the command is completed. not sure if it is somehow possible to make it more realtime/dynamic.

Reply:
Well, this is the already existing code and just for backwards compatibility :)